backend/src/tokenBridge/TokenBridgeContractClient.ts

```diff
@@ -130,4 +130,20 @@ export class TokenBridgeContractClinet implements Injectable {
         const p = await this.instance(network).methods.liquidity(token, userAddress).call();
         return p.toString();
     }
+
+    async getTokenAllowance(userAddress: string, currency: string): Promise<string> {
+        const [network, token] = Helper.parseCurrency(currency);
+        const p = await this.instance(network).methods.getAllowance(userAddress, userAddress).call();
+        return p.toString();
+    }
+
+    async getAvaialableLiquidity(currency: string){
+        const [network, token] = Helper.parseCurrency(currency);
+        
+        const m = await this.instance(network).methods;
+        const ps = await this.helper.erc20(network, token);
+        console.log(network,token,this.contractAddress[network],m,ps);
+        const p = await this.helper.erc20(network, token).methods.balanceOf(this.contractAddress[network]).call();
+        return p.toString();
```

Comment:
use helper.amountToHuman
Always return decimal values from server. Use bigInt values only to communicate with the network